R/plot.R

```diff
@@ -14,3 +14,40 @@ plot.baggr <- function(x, ...) {
   l[["bg"]] <- x
   do.call(baggr_plot, l)
 }
+
+#' Set, get, and replace themes for baggr plots
+#' @rdname baggr_theme_set
+#' @importFrom ggplot2 theme_grey
+#' @importFrom bayesplot theme_default
+#' @param new New theme to use for all baggr plots
+#' @description These functions get, set, and modify the ggplot2 themes
+#' of the baggr plots. By default, these match the bayesplot's package
+#' theme to make it easier to form cohesive graphs across this package
+#' and others.
+#' @return The get method returns the current theme, but all of the
+#' others invisibly return the old theme.
+#' @export
```

Comment:
Needs a couple of examples! I've never used `update` and `replace`, so am not sure. Again, for more details enough to link to other pkgs.

Reply:
I will add them and re-push.